### PR TITLE
Add experimental paint-order property BCD

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1426146

### CODE REVIEW

File: css/properties/paint-order.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/paint-order",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline css/properties/paint-order.json]: @chrisdavidmills It looks like we do support paint-order. https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/css/CSSProperties.json5?l=2067

Comment (chrisdavidmills) [reply css/properties/paint-order.json]: Hrm. Is it behind a pref/flag? I've tried the example at https://developer.mozilla.org/en-US/docs/Web/CSS/paint-order in the latest Canary, and didn't get any joy.

Comment (jpmedley) [reply css/properties/paint-order.json]: Leave it false then. 

Our meta documents are usually pretty reliable indicators of functionality. This is not apparently. I'll dig into this and submit my own PR if need be.

**Comment by Elchi3** [comment]
This is not behind a preference in Firefox https://bugzilla.mozilla.org/show_bug.cgi?id=1435684

**Comment by chrisdavidmills** [comment]
You say that, but when I tested it in the latest nightly, I found that the pref was still set to false. So I decided to include the pref info for now, just in case.

**Comment by chrisdavidmills** [comment]
I'll ask J Kew what the deal is, in the bug.

**Pull request merged**